src/main/java/nextstep/ladder/domain/Ladder.java

```diff
@@ -6,16 +6,21 @@
 
 public class Ladder {
 
-    private List<Line> ladder;
+    private List<LadderLine> ladderLines;
 
     public Ladder(int ladderHeight, int userCount) {
-        RandomMakeStrategy randomMakeStrategy = new RandomMakeStrategy();
-        ladder = Stream.generate(() -> new Line(userCount, randomMakeStrategy)).limit(ladderHeight).collect(Collectors.toList());
+        ladderLines = Stream.generate(() -> LadderLine.init(userCount, new RandomMakeStrategy())).limit(ladderHeight).collect(Collectors.toList());
```

Comment:
```suggestion
        ladderLines = Stream.generate(() -> LadderLine.init(userCount, new RandomMakeStrategy()))
        .limit(ladderHeight)
        .collect(Collectors.toList());
```

스트림은 행위단위로 내려쓰는것이 가독성과 디버깅에 장점이 많은 것 같아요.